### PR TITLE
ATS-804  & ACS-224: Update ATS 1.3.1-A1, T-Core 2.3.2 and SFS with 0.9.0

### PR DESCRIPTION
**Pull request by aymanthefirst**
- transform-router 1.3.1-A1
- transform-core-aio 2.3.2
- shared-file-store 0.9.0

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ services:
 
     transform-router:
         mem_limit: 512m
-        image: quay.io/alfresco/alfresco-transform-router:1.2.0
+        image: quay.io/alfresco/alfresco-transform-router:1.3.0-A1
```
Comment (PunchedChimera) [inline docker-compose/docker-compose.yml:59]:
```suggestion
        image: quay.io/alfresco/alfresco-transform-router:1.3.1-A1
```

File: helm/alfresco-content-services/values.yaml [hunk 2]
```diff
@@ -68,7 +68,7 @@ transformrouter:
   replicaCount: 2
   image:
     repository: quay.io/alfresco/alfresco-transform-router
-    tag: "1.2.0"
+    tag: "1.3.0-A1"
```
Comment (PunchedChimera) [inline helm/alfresco-content-services/values.yaml:71]:
```suggestion
    tag: "1.3.1-A1"
```

File: helm/alfresco-content-services/values.yaml [hunk 3]
```diff
@@ -68,7 +68,7 @@ transformrouter:
   replicaCount: 2
```
Comment (montgolfiere) [inline helm/alfresco-content-services/values.yaml:68]: @aymanthefirst Also the x5 T-Engines need to be bumped from 2.3.1 to 2.3.2 (to be consistent with docker-compose) ?

Comment (aymanthefirst) [reply helm/alfresco-content-services/values.yaml:68]: I changed them all in this helm values file, and also in alfresco-bat/docker-compose.yaml https://github.com/Alfresco/alfresco-bat/pull/6/commits/ae4db73c1ef54166ca773e647d2364cf491163d1

**Pull request merged**